reference/manual-upgrade.md

```diff
@@ -0,0 +1,391 @@
+    /openebs.io/mayastor/apis/v0/clusters/ce05eb25-50cc-400a-a57f-37e6a5ed9bef/namespaces/mayastor/VolumeSpec/bf207797-b23d-447a-8d3f-98d378acfa8a
+    {"uuid":"bf207797-b23d-447a-8d3f-98d378acfa8a","size":1073741824,"labels":{"local":"true"},"num_replicas":3,"status":{"Created":"Online"},"policy":{"self_heal":true},"topology":{"node":{"Explicit":{"allowed_nodes":["worker-1","worker-2","master","worker-0"],"preferred_nodes":["worker-2","master","worker-0","worker-1"]}},"pool":{"Labelled":{"exclusion":{},"inclusion":{"openebs.io/created-by":"operator-diskpool"}}}},"last_nexus_id":"069feb5e-ec65-4e97-b094-99262dfc9f44","operation":null,"thin":false,"target":{"node":"worker-0","nexus":"069feb5e-ec65-4e97-b094-99262dfc9f44","protocol":"nvmf","active":true,"config":{"controllerIdRange":{"start":1,"end":65519},"reservationKey":1,"reservationType":"ExclusiveAccess","preemptPolicy":"Holder"},"frontend":{"host_acl":[]}},"publish_context":null,"volume_group":null}
+    069feb5e-ec65-4e97-b094-99262dfc9f44
+    {"children":[{"healthy":true,"uuid":"f65d9888-7699-4c44-8ee2-f6aaa58dead0"},{"healthy":true,"uuid":"8929e13f-99c0-4830-bcc2-d4b12a541b97"},{"healthy":true,"uuid":"9455811d-480e-4522-b94a-4352ba65cb73"}],"clean_shutdown":false}
+    I have no name!@mayastor-etcd-0:/opt/bitnami/etcd$
+    ```
+    {% endtab %}
+    {% endtabs %}
+
+6. Once all the components have been upgraded, the HA module can now be enabled via the helm upgrade command.  
+    {% tabs %}
+    {% tab title="Command to check the helm list" %}
+    ```text
+    helm upgrade --install mayastor . -n mayastor --set etcd.persistence.storageClass="manual" --set loki-stack.loki.persistence.storageClassName="manual" --set agents.ha.enabled="true"
+    ```
+    {% endtab %}
+    {% tab title="Sample Output" %}
+    ```text
+    Release "mayastor" does not exist. Installing it now.
+    NAME: mayastor
+    LAST DEPLOYED: Tue Apr 25 19:20:53 2023
+    NAMESPACE: mayastor
+    STATUS: deployed
+    REVISION: 1
+    NOTES:
+    OpenEBS Mayastor has been installed. Check its status by running:
+    $ kubectl get pods -n mayastor
+    ```
+    {% endtab %}
+    {% endtabs %}
+
+7. This concludes the process of manual upgrade. Run the below commands to verify the upgrade,
+    {% tabs %}
+    {% tab title="Command to check the helm list" %}
+    ```text
+    helm list -n mayastor
+    ```
+    {% endtab %} 
+    {% tab title="Sample Output" %}
+    ```text
+    NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                           APP VERSION
+    mayastor        mayastor        1               2023-04-25 19:20:53.43928058 +0000 UTC  deployed        mayastor-2.1.0                  2.1.0
+    ```
+    {% endtab %}
+    {% endtabs %}
+
+    {% tabs %}
+    {% tab title="Command to check the status of pods" %}
+    ```text
+    kubectl get pods -n mayastor
```

Comment:
-n mayastor is correct?

Reply:
Yes, because i believe we just want to view the mayastor related pods (which is where we'd installed it initially in the doc) . cc @datacore-tilangovan, Is that correct?

Reply:
Yes, this is to check ha pods deployment status which is in mayastor namespace. But again if the user have any other namespace that goes in here…